osgar/drivers/lora.py

```diff
@@ -19,6 +19,13 @@
   Example:
      b'2|5|4|hello' - original message "hello" was sent by device ID=4 and retransmitted
                       by device ID=5 and retransmitted again by device ID=2
+
+  I/O:
+  This driver can now parse commands for particular device in the form:
+    <device_id>:<cmd>:<hash>
+  The <device_id> has to match in order to publish <cmd>. <hash> is used only for distiniction
+  of messages, and the plan is to use seconds since control center start.
```

Comment:
Has to match what? Can you add an example?

Reply:
So if `device_id` in the message received equals to `self.device_id` the command has found its destination and is published on `cmd` output pin? Otherwise it is resent? Is any id prepending going on during resend? How does the message look like after being resent?

Reply:
All messages are resent if the device did not already send this message.

Yes, only matching `device_id` is then published on `cmd`